packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -257,6 +257,7 @@ watch(
 				start: 0,
 				end: state.endTime
 			},
+			loggingStepSize: state.endTime / state.numberOfTimepoints,
```

Comment:
These logginStepSize seem to be different than that mentioned in the issue this PR is linked to, which is:
```
(EndTime - StartTime) / NumberOfTimepoints
```

Can we double check what it is we weant?

Reply:
Also can `loggingStepSize` be fractional? If so it might mess up some of our dataset parsing which assumes that these are whole numbers 1, 2, 3 ... etc